windows/autobuild/autobuild_windowsinstaller_2_build.ps1

```diff
@@ -0,0 +1,21 @@
+# Sets up the environment for autobuild on Windows
+
+# Get the source path via parameter
+param (
+	[string] $jamulus_project_path = $Env:jamulus_project_path
+)
+# Sanity check of parameters
+if(($jamulus_project_path -eq $null) -or ($jamulus_project_path -eq "")){
+	throw "expecting ""jamulus_project_path"" as parameter or ENV"
+}elseif (!(Test-Path -Path $jamulus_project_path)){
```

Comment:
could you please add spaces here and stick to the code style of the old .ps1 files?

Reply:
What you are saying is basically "use 4 spaces for indent instead of 1 tab", correct? => can do, probably for all ps & sh-files?
Is there anything else in terms of "code-style" I'm not seeing?

Reply:
Personally, I'd suggest two spaces, but corrados used four and I think we should stick to that. Also in the deploy_windows.ps1 file the style is different:

e.g.: 
```powershell
    if (Test-Path -Path $BuildPath) { Remove-Item -Path $BuildPath -Recurse -Force }
```

So you'd need to add a few more spaces here. 

Yes, in a few spaces the code style could be changed. E.g. all variables in the bash scripts, especially those who represent a path should be quoted: ${VAR} should be "${VAR}".

Reply:
Its good to stick to established style. (My personal preference is tabs, then everyone can set their editor to show them what they like. and one indent=one character.)
I did what I saw to be consistent. If theres something I'm missing, tell me.

Does the quoting of path variable concern Powershell only, of bash, too?

Reply:
Yes. Bash is important too.